fission-web-server/library/Fission/Web/Server/Error/Class.hs

```diff
@@ -165,16 +163,16 @@ instance ToServerError JWT.Signature.Error where
 instance ToServerError JWT.Proof.Error where
   toServerError = \case
     ResolverError err     -> toServerError err
-    ScopeOutOfBounds      -> err401 { errBody = displayLazyBS ScopeOutOfBounds      }
-    PotencyEscelation     -> err401 { errBody = displayLazyBS PotencyEscelation     }
+    ScopeOutOfBounds      -> err422 { errBody = displayLazyBS ScopeOutOfBounds      }
+    PotencyEscelation     -> err422 { errBody = displayLazyBS PotencyEscelation     }
     TimeNotSubset         -> err422 { errBody = displayLazyBS TimeNotSubset         }
     MissingExpectedFact   -> err422 { errBody = displayLazyBS MissingExpectedFact   }
     InvalidSignatureChain -> err422 { errBody = displayLazyBS InvalidSignatureChain }
```

Comment:
please tell me you chuckle every time you typed `displayLazyBS`... no?

Reply:
hahaha I totally do!